This doesn't seem to help on macOS, but i'm not sure what is going wrong...

bleed (fails with expected error):
```
  OpenRA.PostProcess -> /Users/paul/src/Open RA/OpenRA.PostProcess.exe
  OpenRA.Game -> /Users/paul/src/Open RA/OpenRA.Game.exe
  /var/folders/gm/2m3hrbv5637f4dhttxn62s1w0000gn/T/tmpf0d03cb3bde64fc3a1069c59af5e1785.exec.cmd: line 2: /Users/paul/src/Open: No such file or directory
/Users/paul/src/Open RA/OpenRA.Game/OpenRA.Game.csproj(79,5): error MSB3073: The command "/Users/paul/src/Open RA/OpenRA.PostProcess.exe /Users/paul/src/Open RA/OpenRA.Game.exe -LAA" exited with code 127.
```

This PR (fails with unexpected error):
```
paul@Ragnarok:Open RA+pr/16718$ make
  OpenRA.PostProcess -> /Users/paul/src/Open RA/OpenRA.PostProcess.exe
  OpenRA.Game -> /Users/paul/src/Open RA/OpenRA.Game.exe
  /var/folders/gm/2m3hrbv5637f4dhttxn62s1w0000gn/T/tmpd6a16d00f0ef4f5ab05f439012066080.exec.cmd: line 2: /Users/paul/src/Open RA/OpenRA.PostProcess.exe: Permission denied
/Users/paul/src/Open RA/OpenRA.Game/OpenRA.Game.csproj(79,5): error MSB3073: The command ""/Users/paul/src/Open RA/OpenRA.PostProcess.exe" "/Users/paul/src/Open RA/OpenRA.Game.exe" -LAA" exited with code 126.
```

--- a/OpenRA.Game/OpenRA.Game.csproj
+++ b/OpenRA.Game/OpenRA.Game.csproj
@@ -17,6 +17,7 @@
     <EnableDefaultCompileItems>false</EnableDefaultCompileItems>
     <CodeAnalysisRuleSet>..\OpenRA.ruleset</CodeAnalysisRuleSet>
     <Configurations>Release;Debug;Release-x86</Configurations>
+    <ManagedExeLauncher Condition="('$(OS)' != 'Windows_NT')">mono </ManagedExeLauncher>
   </PropertyGroup>
   <ItemGroup>
     <!-- Work around an issue where Rider does not detect files in the project root using the default glob -->
@@ -76,7 +77,7 @@
     </ItemGroup>
   </Target>
   <Target Name="PostProcess" AfterTargets="Build" Inputs="$(TargetPath)" Outputs="$(IntermediateOutputPath)\$(TargetFileName).processed">
-    <Exec Command="$(TargetDir)/OpenRA.PostProcess.exe $(TargetPath) -LAA" />
+    <Exec Command="$(ManagedExeLauncher)&quot;$(TargetDir)OpenRA.PostProcess.exe&quot; &quot;$(TargetPath)&quot; -LAA" />
     <Touch Files="$(IntermediateOutputPath)\$(TargetFileName).processed" AlwaysCreate="true" />
   </Target>
 </Project>

--- a/OpenRA.Server/OpenRA.Server.csproj
+++ b/OpenRA.Server/OpenRA.Server.csproj
@@ -15,6 +15,7 @@
     <EnableDefaultCompileItems>false</EnableDefaultCompileItems>
     <CodeAnalysisRuleSet>..\OpenRA.ruleset</CodeAnalysisRuleSet>
     <Configurations>Release;Debug;Release-x86</Configurations>
+    <ManagedExeLauncher Condition="('$(OS)' != 'Windows_NT')">mono </ManagedExeLauncher>
   </PropertyGroup>
   <ItemGroup>
     <!-- Work around an issue where Rider does not detect files in the project root using the default glob -->
@@ -42,7 +43,7 @@
     </ItemGroup>
   </Target>
   <Target Name="PostProcess" AfterTargets="Build" Inputs="$(TargetPath)" Outputs="$(IntermediateOutputPath)\$(TargetFileName).processed">
-    <Exec Command="$(TargetDir)/OpenRA.PostProcess.exe $(TargetPath) -LAA" />
+    <Exec Command="$(ManagedExeLauncher)&quot;$(TargetDir)OpenRA.PostProcess.exe&quot; &quot;$(TargetPath)&quot; -LAA" />
     <Touch Files="$(IntermediateOutputPath)\$(TargetFileName).processed" AlwaysCreate="true" />
   </Target>
 </Project>

--- a/OpenRA.Utility/OpenRA.Utility.csproj
+++ b/OpenRA.Utility/OpenRA.Utility.csproj
@@ -15,6 +15,7 @@
     <EnableDefaultCompileItems>false</EnableDefaultCompileItems>
     <CodeAnalysisRuleSet>..\OpenRA.ruleset</CodeAnalysisRuleSet>
     <Configurations>Release;Debug;Release-x86</Configurations>
+    <ManagedExeLauncher Condition="('$(OS)' != 'Windows_NT')">mono </ManagedExeLauncher>
   </PropertyGroup>
   <ItemGroup>
     <!-- Work around an issue where Rider does not detect files in the project root using the default glob -->
@@ -42,7 +43,7 @@
     </ItemGroup>
   </Target>
   <Target Name="PostProcess" AfterTargets="Build" Inputs="$(TargetPath)" Outputs="$(IntermediateOutputPath)\$(TargetFileName).processed">
-    <Exec Command="$(TargetDir)/OpenRA.PostProcess.exe $(TargetPath) -LAA" />
+    <Exec Command="$(ManagedExeLauncher)&quot;$(TargetDir)OpenRA.PostProcess.exe&quot; &quot;$(TargetPath)&quot; -LAA" />
     <Touch Files="$(IntermediateOutputPath)\$(TargetFileName).processed" AlwaysCreate="true" />
   </Target>
 </Project>

